build.py

```diff
@@ -93,6 +93,7 @@ def do_post_retry(url=None, data=None, headers=None, files=None):
             time.sleep(10)
 
 def do_make(target=None, log=False):
+    os.environ['KBUILD_BUILD_USER'] = 'KernelCI'
```

Comment:
Can you please explain what this is all about?  Maybe split it into a separate commit.

Reply:
I put this in the same commit as it's all related. A kernel build will try to use `whoami` to give the "root@buildslave" name to the kernel, however in a docker container that doesnt work, so instead you can override this build environment to set it.
See https://github.com/kernelci/kernelci-build/pull/51#pullrequestreview-148291953